packages/moonstone/Scroller/Scroller.js

```diff
@@ -343,30 +345,30 @@ class ScrollerBase extends Component {
 
 	scrollToBoundary = (direction) => {
 		const
-			{scrollBounds, scrollPos} = this.uiRef,
+			{scrollBounds, scrollPos} = this.uiRef.current,
 			isVerticalDirection = (direction === 'up' || direction === 'down');
 
 		if (isVerticalDirection) {
 			if (scrollPos.top > 0 && scrollPos.top < scrollBounds.maxTop) {
-				this.uiRef.props.cbScrollTo({align: direction === 'up' ? 'top' : 'bottom'});
+				this.uiRef.current.props.cbScrollTo({align: direction === 'up' ? 'top' : 'bottom'});
 			}
 		} else if (scrollPos.left > 0 && scrollPos.left < scrollBounds.maxLeft) {
-			this.uiRef.props.cbScrollTo({align: this.props.rtl ? reverseDirections[direction] : direction});
+			this.uiRef.current.props.cbScrollTo({align: this.props.rtl ? reverseDirections[direction] : direction});
 		}
 	}
 
 	handleLeaveContainer = ({direction, target}) => {
-		const contentsContainer = this.uiRef.containerRef;
+		const contentsContainer = this.uiRef.current.containerRef.current;
 		// ensure we only scroll to boundary from the contents and not a scroll button which
-		// lie outside of this.uiRef.containerRef but within the spotlight container
+		// lie outside of this.uiRef.current.containerRef but within the spotlight container
 		if (contentsContainer && contentsContainer.contains(target)) {
 			this.scrollToBoundary(direction);
 		}
 	}
 
 	initUiRef = (ref) => {
 		if (ref) {
-			this.uiRef = ref;
+			this.uiRef = {current: ref};
```

Comment:
I think we'd better change `uiRef` to another name to prevent any misunderstanding rather than assigning `{current: ref}` to make it look like a result of `forwardRef`.

Reply:
Rename `uiRef` as `uiRefCurrent`. Same as `uiChildRef`.